content/contributing-terms.en.md

```diff
@@ -34,7 +34,7 @@ The service name is exposed to end users. It should reflect as closely as possib
   - _Example: `Firebase` (by Google) → `Firebase`_.
   - _Example: `App Store` (by Apple) → `App Store`_.
 
-> If you have a hard time finding the service name, check out the [practical guidelines to find the service name](declarations-guidelines.md#service-name), and feel free to mention your uncertainties in the pull request! We will help you improve the service name if necessary 🙂
+> If you have a hard time finding the service name, check out the [practical guidelines to find the service name](./guidelines/declaring.en.md#service-name), and feel free to mention your uncertainties in the pull request! We will help you improve the service name if necessary 🙂
```

Comment:
The relative path is incorrect, and you shouldn't include the extension in the filename.
The best way to check the generated links are correct is to generate the documentation locally by following the instructions in the README 🙂 

```suggestion
> If you have a hard time finding the service name, check out the [practical guidelines to find the service name](../guidelines/declaring#service-name), and feel free to mention your uncertainties in the pull request! We will help you improve the service name if necessary 🙂
```

Reply:
Ok I made the requested changes